docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -0,0 +1,145 @@
+# Flex Layout
+
+## Overview
+
+`FlexLayout` is a flexible box layout, providing a more efficient way to lay out, align and distribute space among items in the container, even when their size is unknown or dynamic.
+
+A layout which provides features like wrapping so if items no long fit on an axis they can automatically be positioned on another row or column.
+
+`FlexLayout` defines four properties that affect the size, orientation, and alignment of child views. These properties are described in more detail in the table below.
+Please see each categories below.
+
+| Property               | Type            | Description |
+| -----------------------| --------------- | ------------ |
+| `Direction`            | FlexDirection   | The orientation the flex items are laid out in (Column/Row) |
+| `Justification`        | FlexJustification | Alignment of items along the flex axis when free space available |
+| `Alignment`            | AlignmentType     | Alignment of items along the cross axis when free space available |
+| `WrapType`             | FlexWrapType    | Enable wrapping of items |
+
+
+Natural size of items are used which can be different for each item. And setting the size of an item has no effect.
+
+`Justification` applies to the flex Direction axis whilst `Alignment` is the (other) cross axis. Changing the Direction will apply the `Justification` to the new direction.
+
+
+## FlexDirection
+
+`Direction` property specifies the main axis direction along which flex items are placed.
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/flexDirection.png">
+</td>
+</tr>
+</table>
+
+| Property value |  Description |
+|----------------|--------------|
+| `Row`          | Place items horizontally in a row |
+| `RowReverse`   | Place items horizontally in a row, but in reverse order |
+| `Column`       | Place items vertically in a column |
+| `ColumnReverse` | Place items vertically in a column, but in reverse order |
+
+### Usage
```

Comment:
Suggestion:
To remove this heading as it does not align with the flow of other pages.

Reply:
Done.